src/indieweb_utils/feeds/discovery.py

```diff
@@ -66,4 +81,15 @@ def discover_web_page_feeds(url: str, user_mime_types: Optional[List[str]] = Non
     if h_feed:
         feeds.append(FeedUrl(url=url, mime_type="text/html", title=page_title.text))
 
+    http_headers = _find_links_in_headers(headers=web_page_request.headers, target_headers=["alternate", "feed"])
+
+    for rel, item in http_headers.items():
+        if rel == "feed" or rel == "alternate":
```

Comment:
We shouldn't need to check rel type here because `_find_links_in_headers` should only be returning "feed" and "alternative" headers, right?